core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/organism/drawer/ModalNavigationDrawer.kt

```diff
@@ -19,20 +19,23 @@ fun ModalNavigationDrawer(
     val drawerState = rememberDrawerState(initialValue = DrawerValue.Closed)
     val scope = rememberCoroutineScope()
 
-    val openDrawer = { scope.launch { drawerState.open() } }
-    val closeDrawer = {
```

Comment:
The compilation error due to the change in type inference can be fixed by explicitly specifying the type:

```kotlin
val openDrawer: () -> Unit = //…
val closeDrawer: () -> Unit = //…
```

or

```kotlin
private typealias DrawerAction = () -> Unit
```

and
```kotlin
val openDrawer: DrawerAction = //…
val closeDrawer: DrawerAction = //…
```

That way the code remains more readable than inlining the lambdas. See https://github.com/thunderbird/thunderbird-android/pull/7769#discussion_r1574935014